setup-machine.sh

```diff
@@ -200,7 +196,7 @@ sudo cp config/apparmor.d/usr.sbin.cups-browsed /etc/apparmor.d/
 
 
 # let vx-services scan
-if [ "${CHOICE}" != "vx-mark" ] && [ "${CHOICE}" != "bas" ] 
+if [ "${CHOICE}" != "vx-mark" ]
```

Comment:
@benadida should this also include a skip for VxAdmin? Its enabled scanning functions.

Reply:
yeah this was left open for when we thought maybe vxadmin would scan ballots, too, safe to exclude it for now.